crates/bevy_asset/src/io/source.rs

```diff
@@ -486,7 +486,7 @@ impl AssetSource {
                     sender,
                     file_debounce_wait_time,
                 )
-                .unwrap(),
+                .unwrap_or_else(|e| panic!("Failed to create file watcher: {}", e)),
```

Comment:
This does the same, but is shorter <https://doc.rust-lang.org/stable/std/result/enum.Result.html#method.expect>
```suggestion
                .expect("Failed to create file watcher"),
```

Reply:
thanks!